src/EpisodeManagementService/UpdateEpisode/Episode.cs

```diff
@@ -0,0 +1,4 @@
+public class Episode
+{
+  public string episode_id { get; set; }
```

Comment:
Could we use PascalCase for the property name, please

Reply:
sure